### PR TITLE
utility to download and install sdk

### PR DESCRIPTION
**Pull request by danrubel**
@devoncarew

### CODE REVIEW

File: lib/plugin.dart [hunk 1]
```diff
@@ -191,6 +192,7 @@ class AtomDartPackage extends AtomPackage {
       });
     });
     _addCmd('atom-workspace', 'dartlang:send-feedback', (_) => _handleSendFeedback());
+    _addCmd('atom-workspace', 'dartino:installSdk', dartino.promptInstallSdk);
```
Comment (devoncarew) [inline lib/plugin.dart]: 'dartino:install-sdk'?

**Comment by devoncarew** [comment]
Lgtm

I couldn't see - does this clean up the temp dir and SDK zip?

File: lib/plugin.dart (hunk 1 above)
Comment (danrubel) [inline lib/plugin.dart]: Good idea. Done.

**Comment by danrubel** [comment]
It should but does not. Fixed.

**Comment by danrubel** [comment]
Merged

**Pull request closed without merging**